src/app/Recordings/ArchivedRecordingsTable.tsx

```diff
@@ -554,6 +579,16 @@ export const ArchivedRecordingsTable: React.FunctionComponent<ArchivedRecordings
     </Drawer>
   );
 };
+
+export interface RecordingRowProps {
+  key: string;
+  recording: any;
```

Comment:
This should be `ArchivedRecording` type. Also, on this issue, could you also change this field type to `Recording` instead? I must have missed it from the previous PR.

https://github.com/cryostatio/cryostat-web/blob/770ace96fd8654e1c56c4c2f84dc05b9d888d99c/src/app/Recordings/RecordingActions.tsx#L57

Reply:
Ohh yeh sorry I meant only change the type in `RecordingActions` to `Recording`.  And the type in props should be `ArchivedRecording`. Should've reference the link first.